sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -0,0 +1,322 @@
+
+### Input
+
+A **text element** (`string`), is a single unit of input to be processed by the translation models in the Translator service. Operations on `TextTranslationClient` may take a single text element or a collection of text elements.
+For text element length limits, maximum requests size, and supported text encoding see [here][translator_limits].
+
+## Examples
+
+The following section provides several code snippets using the `client` [created above](#create-a-texttranslationclient-using-an-api-key-and-region-credential), and covers the main features present in this client library. Although most of the snippets below make use of asynchronous service calls, keep in mind that the Text Translation for Python library package supports both synchronous and asynchronous APIs.
+
+### Get Supported Languages
+
+Gets the set of languages currently supported by other operations of the Translator.
+
+```python
+try:
+    response = response = text_translator.get_languages()
+
+    print(f"Number of supported languages for translate operation: {len(response.translation) if response.translation is not None else 0}")
+    print(f"Number of supported languages for transliterate operation: {len(response.transliteration) if response.transliteration is not None else 0}")
+    print(f"Number of supported languages for dictionary operations: {len(response.dictionary) if response.dictionary is not None else 0}")
+
+    if response.translation is not None:
+        print("Translation Languages:")
+        for key, value in response.translation.items():
+            print(f"{key} -- name: {value.name} ({value.native_name})")
+
+    if response.transliteration is not None:
+        print("Transliteration Languages:")
+        for key, value in response.transliteration.items():
+            print(f"{key} -- name: {value.name}, supported script count: {len(value.scripts)}")
+
+    if response.dictionary is not None:
+        print("Dictionary Languages:")
+        for key, value in response.dictionary.items():
+            print(f"{key} -- name: {value.name}, supported target languages count: {len(value.translations)}")
+
+except HttpResponseError as exception:
+    print(f"Error Code: {exception.error_code}")
+    print(f"Message: {exception.message}")
+```
+
+For samples on using the `languages` endpoint refer to more samples [here][languages_sample].
+
+Please refer to the service documentation for a conceptual discussion of [languages][languages_doc].
+
+### Translate
+
+Renders single source-language text to multiple target-language texts with a single request.
+
+```python
+try:
+    source_language = "en"
+    target_languages = ["cs"]
+    input_text_elements = [ InputTextItem(text = "This is a test") ]
+
+    response = text_translator.translate(content = input_text_elements, to = target_languages, from_parameter = source_language)
+    translation = response[0] if response else None
```

Comment:
Our service doesn't have notion of partial translations. It is all or nothing. In case of any error, the body looks like:
```json
{
  "error": {
    "code": 400036,
    "message": "The target language is not valid."
  }
}
```

Reply:
Got it - thanks